### PR TITLE
Fix rms/output folders to be kept with opt. 3

### PR DESCRIPTION
**Pull request by jcrivenaes**
For fmu_copy_revision, refer to issue #450

### CODE REVIEW

**Comment by ertomatic** [comment]
Can one of the admins verify this patch?

File: src/subscript/fmu_copy_revision/fmu_copy_revision.py [hunk 1]
```diff
@@ -35,6 +35,10 @@
 
     fmu_copy_revision --source 21.0.0  (...other options are defaulted)
 
+Backup:
```
Comment (berland) [inline src/subscript/fmu_copy_revision/fmu_copy_revision.py:38]: Should we include this? It is maybe better to get users to report errors that can be fixed?

File: tests/test_fmu_copy_revision.py [hunk 2]
```diff
@@ -125,6 +126,42 @@ def test_rsync_profile3(datatree):
...
+    runner.profile = 4
+    runner.source = source
+    runner.construct_target(target)
+    runner.define_filterpattern()
+    print(runner.filter)
+    runner.do_rsyncing()
+
+    assert (datatree / target / "rms" / "input" / "faults" / "f1.dat").exists()
+
+    # profile 4: rms/output folders shall will be empty as in option 3 and
```
Comment (berland) [inline tests/test_fmu_copy_revision.py:157]: "shall will" -> "will" ?

File: tests/test_fmu_copy_revision.py [hunk 3]
```diff
@@ -125,6 +126,42 @@ def test_rsync_profile3(datatree):
...
+    # hence removed accoridng to option 4
```
Comment (berland) [inline tests/test_fmu_copy_revision.py:158]: accoridng -> according

File: tests/test_fmu_copy_revision.py [hunk 4]
```diff
@@ -234,8 +271,30 @@ def test_choice_profile3(datatree):
...
+    )
+    print(result.stdout.decode())
+
+    assert "Sync files using multiple threads" in result.stdout.decode()
+    assert target in result.stdout.decode()
+    assert (datatree / target / "rms" / "input" / "faults" / "f1.dat").exists()
+    assert not (datatree / target / "rms" / "output").exists()
     assert not (datatree / target / "rms" / "output" / "any_out.dat").exists()
     assert not (datatree / target / "rms" / "input" / "faults" / "x.dat").exists()
+    assert (datatree / target / "rms" / "input" / "faults").exists()
```
Comment (berland) [inline tests/test_fmu_copy_revision.py:297]: (this test is a noop, as the presence of "f1.dat" is tested above)

File: src/subscript/fmu_copy_revision/fmu_copy_revision.py [hunk 5]
```diff
@@ -140,7 +144,6 @@
 - ert/output/
 - ert/storage/
 - ert/**/storage/
-- rms/output
```
Comment (berland) [inline src/subscript/fmu_copy_revision/fmu_copy_revision.py:143]: Could the documentation line further up more clearly state what it means to have an entry in DIRFILTER<int> ?

File: src/subscript/fmu_copy_revision/fmu_copy_revision.py (hunk 1 above)
Comment (jcrivenaes) [reply src/subscript/fmu_copy_revision/fmu_copy_revision.py:38]: Yes the intention was just if the user got panic. I can reformulate a bit

File: src/subscript/fmu_copy_revision/fmu_copy_revision.py (hunk 5 above)
Comment (jcrivenaes) [reply src/subscript/fmu_copy_revision/fmu_copy_revision.py:143]: Yes, certainly needed. When doing that more errors and imprecise wording were revealed, so there will come additional changes

File: src/subscript/fmu_copy_revision/fmu_copy_revision.py [hunk 6]
```diff
@@ -35,9 +35,14 @@
...
     fmu_copy_revision --source 21.0.0  (...other options are defaulted)
 
-Backup:
-    This script is rewritten. Use fmu_copy_revision_old and report issues if something
-    seems wrong!
+Note:
+    This script is fully rewritten Q3/Q4 2021. If you encounter a problem, then
+
+      a) Report the issue to https://github.com/equinor/subscript/issues
+         or to JRIV@equinor.com so we can improve
```
Comment (berland) [inline src/subscript/fmu_copy_revision/fmu_copy_revision.py:42]: yammer instead of email?

**Review by berland — APPROVED** [review]

File: src/subscript/fmu_copy_revision/fmu_copy_revision.py (hunk 6 above)
Comment (jcrivenaes) [reply src/subscript/fmu_copy_revision/fmu_copy_revision.py]: Good point, will modify

**Pull request merged**